services/core-api/app/api/now_applications/models/now_application_document_xref.py

```diff
@@ -11,27 +12,31 @@ class NOWApplicationDocumentXref(AuditMixin, Base):
     __tablename__ = "now_application_document_xref"
     _edit_groups = [NOW_APPLICATION_EDIT_GROUP]
 
-    now_application_document_xref_guid = db.Column(UUID(as_uuid=True),
-                                                   primary_key=True,
-                                                   server_default=FetchedValue())
-    mine_document_guid = db.Column(UUID(as_uuid=True),
-                                   db.ForeignKey('mine_document.mine_document_guid'))
-    now_application_id = db.Column(db.Integer,
-                                   db.ForeignKey('now_application.now_application_id'),
-                                   server_default=FetchedValue())
+    now_application_document_xref_guid = db.Column(
+        UUID(as_uuid=True), primary_key=True, server_default=FetchedValue())
+    mine_document_guid = db.Column(
+        UUID(as_uuid=True), db.ForeignKey('mine_document.mine_document_guid'))
+    now_application_id = db.Column(
+        db.Integer,
+        db.ForeignKey('now_application.now_application_id'),
+        server_default=FetchedValue())
     now_application_document_type_code = db.Column(
         db.String,
         db.ForeignKey('now_application_document_type.now_application_document_type_code'))
 
     description = db.Column(db.String)
     is_final_package = db.Column(db.Boolean)
 
+    now_application_review_id = db.Column(
+        db.Integer, db.ForeignKey('now_application_review.now_application_review_id'))
     # NOWApplicationDocumentType
     now_application_document_type = db.relationship('NOWApplicationDocumentType', lazy='joined')
-    now_application_document_type_code_description = association_proxy('now_application_document_type', 'description')
-
+    now_application = db.relationship('NOWApplication', lazy='select')
     # MineDocument
-    mine_document = db.relationship('MineDocument', lazy='joined')
+    mine_document = db.relationship(
```

Comment:
Not really part of this PR, but `mine_document` seems like a strange name.

Reply:
yes it does. I'm hoping MDS-2578 deals with it.